### PR TITLE
Release v1.3

### PR DESCRIPTION
**Pull request by xNombre**

### CODE REVIEW

**Comment by xNombre** [comment]
Ive added a rule about `ro.control_privapp_permissions` please confirm if your phone boots fine without it.

File: README.md
Comment (DarkJoker360) [inline README.md]: typo at "mush"

Comment (xNombre) [reply README.md]: Fixed.

File: README.md
Comment (gwolf2u) [inline README.md:14]: How to validate?

File: README.md
Comment (gwolf2u) [inline README.md:68]: keep if possible for now as support is getting dropped with A13

File: README.md
Comment (gwolf2u) [inline README.md]: suggest to drop this
we can't validate and does less work for us to maintain gapps

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: README.md
Comment (xNombre) [reply README.md]: As long as @neobuddy89 wants to maintain our gapps I would keep it.

Comment (gwolf2u) [reply README.md]: we done even have a 12.1 branch
as I said, less work, better for us
https://gitlab.com/crdroidandroid/android_vendor_gapps

File: README.md
Comment (xNombre) [reply README.md:14]: We can't validate any of these points as maintainers are building on their own computers, we just have to trust them :)
Also, from time to time we should check a few devices chosen randomly from ota repository and "punish" maintainers if needed.

File: README.md
Comment (xNombre) [reply README.md]: Im waiting for Pranav's answer, if he doesnt want to maintain it I will change this rule to enforce the most basic variant of gapps.

File: README.md
Comment (xNombre) [reply README.md:68]: It is not needed as per 'should' is already a suggestion rather than an enforcement.

File: README.md
Comment (xNombre) [reply README.md]: Removed our gapps reference.

**Review by gwolf2u — APPROVED** [review]

**Comment by gwolf2u** [comment]
OK from me
Squash merge

**Pull request merged**
